src/game/system/_private/PhysicsSystem.cpp

```diff
@@ -83,7 +83,7 @@ void PhysicsSystem::Tick(deltaTime_t dt)
 			bodyStructure->body->setLinearVelocity(btVector3(0.5, bodyStructure->body->getLinearVelocity().y(), bodyStructure->body->getLinearVelocity().z()));
 		}
 
-		m_pWorld->stepSimulation(dt);
+		m_pWorld->stepSimulation(1.f/ 25000.f);
```

Comment:
This doesn't really work as intended. This performs physics simulations with fixed steps, but the physics system itself is ticked according to a varying timestep. What this means is that a faster computer will lead to a faster game (i.e. this constant is only suitable for a particular build on a particular machine)